games/game_of_life.c

```diff
@@ -0,0 +1,146 @@
+/**
+ * @file
+ * @brief [Game of life](https://en.wikipedia.org/wiki/Conway%27s_Game_of_Life)
+ * implementation
+ *
+ * @details The Game of Life, also known simply as Life, is a cellular automaton devised by the British
+ * mathematician John Horton Conway in 1970. It is a zero-player game, meaning that its evolution is determined
+ * by its initial state, requiring no further input. One interacts with the Game of Life by creating an initial
+ * configuration and observing how it evolves. It is Turing complete and can simulate a universal constructor or any other Turing machine.
+ * @author [Ali Naderi](https://github.com/khod-naderi)
+*/
+
+#include <stdio.h> // stundard input/output
+#include <stdlib.h> // for genrate random number AND memory management 
+#include <time.h> // get time system for seed random number
+#include <unistd.h> // for sleep func (usleep)
+
+// define Make fixed values
+#define MAX_X 60 // End point X
+#define MAX_Y 20 // End point Y
+
+
+/**
+ * draw program function. (void)
+ * For Show the world
```

Comment:
Please add proper documentation. See #797 as a reference. Check other parts of the code, too.

Reply:
Still not fixed.